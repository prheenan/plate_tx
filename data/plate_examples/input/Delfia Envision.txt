READER_NAME	READER_DESC	FILE_FRAGMENT
Delfia Envision 96 Max Rlu		"Plate information
Plate,Repeat,Barcode,Measured height,Inside temperature at start,Inside temperature at end,Humidity at start,Humidity at end,Ambient temperature at start,Ambient temperature at end,Group,Label,ScanX,ScanY,Measinfo,Kinetics,Measurement date,
1,1,,14.04,27.15,27.17,27.7,27.7,26.84,26.87,1,XXXXX-Fluo-Tryp(1),0,0,De=1st Ex=Top Em=Top Wdw=N/A,0,4/21/2015 3:59:01 PM,

Background information
Plate,Label,Result,Signal,Flashes/Time,Meastime,MeasInfo,
1,XXXXX-Fluo-Tryp,0,70330,40,00:00:00.000,De=1st Ex=Top Em=Top Wdw=N/A,

Results for XXXXX-Fluo-Tryp(1) - channel 1 (RFU)
,01,02,03,04,05,06,07,08,09,10,11,12,
A,120364,112997,96192,63676,42611,37757,38816,34656,32367,30728,36832,35976,
B,40654,40021,37777,38635,35754,36049,39480,37621,35101,35465,35359,37946,
C,37044,36700,36007,36441,36984,37822,34305,33228,33733,34981,40471,34895,
D,33242,32695,37256,35466,35450,32598,55780,39837,39440,39320,36931,34792,
E,35155,36861,38018,45814,38628,44583,49678,53145,44156,43237,34188,37189,
F,35627,39186,44148,46754,35501,51360,43468,51063,36422,38541,36131,36128,
G,35774,36655,39092,40596,43009,47889,48112,47066,48407,35620,38995,33546,
H,36187,36908,39213,32616,33534,34742,32536,34515,34647,29752,34682,34339,


Basic assay information
Assay ID: ,,,,11087
Assay Started: ,,,,4/21/2015 3:58:31 PM
Assay Finished: ,,,,4/21/2015 3:59:08 PM
Assay Exported: ,,,,4/21/2015 3:59:08 PM
Protocol ID: ,,,,100256
Protocol Name: ,,,,XXXXX-Tryp
Serial#: ,,,,1030142



Protocol information
Protocol:
Protocol name,,,,XXXXX-Tryp
Number of assay repeats,,,,1
Start assay repeat each,,,,N/A
Number of plate repeats,,,,1
Start plate repeat each,,,,N/A
Is label meas. height used,,,,Yes
Height of measurement,,,,Defined in label
Is gripper height used,,,,No
Mode of measurement,,,,By Rows  bi-directional
Rotated plate,,,,No
Soft move,,,,No
Protocol notes,,,,

Plate type:
Name of the plate type,,,,96 General
Number of rows,,,,8
Number of columns,,,,12
Number of the wells in the plate,,,,96
Height of the plate,,,,14.35 mm

Coordinates of corners:
96 General
14.38 mm x------------------------------------------------------ x 113.38 mm
11.24 mm                                                           11.24 mm

14.38 mm x------------------------------------------------------ x 113.38 mm
74.24 mm                                                           74.24 mm

Platemap:
Plate,,,,1
Group,,,,1

,01,02,03,04,05,06,07,08,09,10,11,12,
A,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
B,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
C,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
D,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
E,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
F,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
G,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-
H,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,- ,-

 - - Undefined

Calculations:
No calculations.

Auto export parameters:
Export format,,,,Plate2
Include basic assay information,,,,Yes
Place assay information at,,,,End of file
Include basic plate information,,,,Yes
Place plate information at,,,,Beginning of plate
Include protocol information,,,,Yes
Protocol info level,,,,Large
Include error and warning information,,,,Yes
Include background information,,,,Yes
Add plate number to the file name,,,,Yes
Each plate to separate file,,,,No
Field separator to use,,,,System
File name format,,,,\\_.csv

Operations:
Plate 1
  Group 1
    Measurement
      Label,,,,XXXXX-Fluo-Tryp(1)

Labels:
XXXXX-Fluo-Tryp,,,,4000051
Top mirror,,,,General
Bottom mirror,,,,N/A
Exc. filter,,,,FITC 485
Using of excitation filter,,,,Top
2nd exc. filter,,,,N/A
Using of 2nd excitation filter,,,,Top
Ems. filter,,,,APC 665
2nd ems. filter,,,,N/A
Measurement height,,,,6 mm
Number of flashes,,,,40
Number of flashes integrated,,,,1
PMT gain,,,,150
Excitation light,,,,100%
Reference excitation light,,,,100%
Emission light,,,,100%
Reference AD gain,,,,2
Reference signal,,,,327706
Last edited,,,,4/21/2015 3:57:00 PM
Last edited by,,,,EnVision
Factory preset,,,,No

Filters:
FITC 485,,,,102
Filter type,,,,Excitation
Description,,,,X485 CWL=485nm BW=14nm Tmin=60%
Used with,,,,DELFIA - Time-resolved Fluorescence Fluorescence Intensity Fluorescence Polarization LANCE - Time-resolved Fluorescence
Last edited,,,,9/22/2009 10:02:15 AM
Last edited by,,,,Installation
Factory preset,,,,Yes

APC 665,,,,205
Filter type,,,,Emission
Description,,,,M665 CWL=665nm BW=7 5nm Tmin=75%
Used with,,,,DELFIA - Time-resolved Fluorescence Fluorescence Intensity Fluorescence Polarization LANCE - Time-resolved FluorescenceLuminescence DELFIA - Time-resolved Fluorescence Fluorescence Intensity LANCE - Time-resolved Fluorescence
Last edited,,,,9/22/2009 10:02:15 AM
Last edited by,,,,Installation
Factory preset,,,,Yes

Mirror modules:
General,,,,401
Used by,,,,Top
Dual,,,,No
Bias,,,,No
Description,,,,BS50
Used with,,,,Luminescence DELFIA - Time-resolved Fluorescence Fluorescence Intensity LANCE - Time-resolved Fluorescence
Last edited,,,,9/22/2009 10:02:15 AM
Last edited by,,,,Installation
Factory preset,,,,Yes

Instrument:
Serial number,,,,1030142
Nickname,,,,EnVision






Exported with EnVision Workstation version 1.12
