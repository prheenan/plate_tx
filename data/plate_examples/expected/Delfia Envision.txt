Row,01,02,03,04,05,06,07,08,09,10,11,12
A,120364.0,112997.0,96192.0,63676.0,42611.0,37757.0,38816.0,34656.0,32367.0,30728.0,36832.0,35976.0
B,40654.0,40021.0,37777.0,38635.0,35754.0,36049.0,39480.0,37621.0,35101.0,35465.0,35359.0,37946.0
C,37044.0,36700.0,36007.0,36441.0,36984.0,37822.0,34305.0,33228.0,33733.0,34981.0,40471.0,34895.0
D,33242.0,32695.0,37256.0,35466.0,35450.0,32598.0,55780.0,39837.0,39440.0,39320.0,36931.0,34792.0
E,35155.0,36861.0,38018.0,45814.0,38628.0,44583.0,49678.0,53145.0,44156.0,43237.0,34188.0,37189.0
F,35627.0,39186.0,44148.0,46754.0,35501.0,51360.0,43468.0,51063.0,36422.0,38541.0,36131.0,36128.0
G,35774.0,36655.0,39092.0,40596.0,43009.0,47889.0,48112.0,47066.0,48407.0,35620.0,38995.0,33546.0
H,36187.0,36908.0,39213.0,32616.0,33534.0,34742.0,32536.0,34515.0,34647.0,29752.0,34682.0,34339.0
